app/helpers/smart_proxies_helper.rb

```diff
@@ -1,5 +1,5 @@
 module SmartProxiesHelper
-  TABBED_FEATURES = ["Puppet","Puppet CA"]
+  TABBED_FEATURES = ["Puppet", "Puppet CA", "Logs"]
```

Comment:
Alphanum sorting unfortunately does not look good:

![image](https://cloud.githubusercontent.com/assets/49752/12718703/909ad5f4-c8f0-11e5-9044-af965706b240.png)

Do you want me to add a priority number to Features?